lnrpc/walletrpc/walletkit.proto

```diff
@@ -286,6 +286,16 @@ message AddrRequest {
     default wallet account is used.
     */
     string account = 1;
+
+    /*
+    The type of address to derive.
```

Comment:
Ah totally forgot about this, guess we've been defaulting to P2WKH, will be useful post taproot as well.